neo/rawio/neuralynxrawio/neuralynxrawio.py

```diff
@@ -132,26 +133,56 @@ class NeuralynxRawIO(BaseRawIO):
     ]
 
     def __init__(
-        self, dirname="", filename="", exclude_filename=None, keep_original_times=False, strict_gap_mode=True, **kargs
+            self,
+            dirname="",
+            filename="",
+            include_filename=None,
+            exclude_filename=None,
+            keep_original_times=False,
+            strict_gap_mode=True,
+            **kargs
     ):
 
+        if (include_filename is not None) and (filename != ""):
+            raise ValueError("filename and include_filenames cannot be both assigned")
+        if (include_filename is None) and (filename == "") and (dirname == ""):
+            raise ValueError("One of dirname or filename or include_files must be provided.")
+
+        if dirname != "" and (include_filename is not None):
+            include_filename = [os.path.join(dirname, f) for f in include_filename]
+            dirname = ""
```

Comment:
I see why you changed dirname to an empty string for the boolean check below, but for the include_filename it should still return the boolean, so why not just split the boolean below

```python
if dirname != "":
    if include_filename is not None:
          self.rawmode = 'multiple-files'
    else:
          self.rawmode='one-dir'
```

for line 160. I wouldn't delete the dirname info since it is actually the root of the current NeuralynxRawIO instance.

Reply:
Yes, that makes sense!